python/graphstorm/inference/emb_infer.py

```diff
@@ -23,45 +23,53 @@
 
 
 class GSgnnEmbGenInferer(GSInferrer):
-    """ Embedding Generation inferrer.
+    """ Inferrer for embedding generation tasks.
 
-    This is a high-level inferrer wrapper that can be used directly
-    to generate embedding for inferer.
+    ``GSgnnEmbGenInferer`` inherits from the ``GSInferrer`` and use the functions
+    provided by ``GSInferrer`` to define the ``infer()`` method that performs one work:
+
+    * Generate node embeddings and save to disk.
 
     Parameters
     ----------
-    model : GSgnnNodeModel
-        The GNN model with different task.
+    model : GSgnnModel
+        This model could be one of the built-in GraphStorm prediction models, i.e.,
+        ``GSgnnNodeModel``, ``GSgnnEdgeModel``, ``GSgnnLinkPredictionModel``, or model
+        classes that inherit them and ``GSgnnNodeModelBase``, ``GSgnnEdgeModelBase``,
+        ``GSgnnLinkPredictionModelBase``.
     """
     def infer(self, data, infer_ntypes, save_embed_path, eval_fanout,
             use_mini_batch_infer=False,
             node_id_mapping_file=None,
             save_embed_format="pytorch",
             infer_batch_size=1024):
-        """ Do Embedding Generating
-
-        Generate node embeddings and save into disk.
+        """ Do Embedding Generating.
```

Comment:
```suggestion
        """ Generate node embeddings and save to disk.
```
"Generating" implies a continuous action